src/db_adapters/accounts.rs

```diff
@@ -62,6 +62,35 @@ pub(crate) async fn handle_accounts(
                         }
                     }
                     near_primitives::views::ActionView::DeleteAccount { .. } => {
+                        let target = schema::access_keys::table.filter(
+                            schema::access_keys::dsl::account_id
+                                .eq(receipt.receiver_id.clone().to_string()),
+                        );
```

Comment:
1. This whole handling should belong to the `access_keys` adapter (not only from the structure point of view, but also to avoid data race between `handle_access_keys` and `handle_accounts` which we run in parallel in `handle_message`)
2. This way of updating the keys that belong to the deleted account is not idempotent (as a rule of thumb, consider a usecase when we have a fully synced database and we run another indexer from genesis: if we have all the accounts and access keys in the latest state and then we start from genesis and somewhere in history the account has been deleted, it does not mean that we want to delete the access keys [further in the history the account could be recreated and new keys added]), we should also take `last_update_block_height` into account.